package.json

```diff
@@ -52,6 +69,7 @@
     "test:testcafe-setup": "mkdir -p test-reports/testcafe && yarn start:mockDuo --no-open",
     "test:testcafe-run": "wait-on http-get://localhost:3000 && testcafe -c 2 chrome:headless test/testcafe/spec/ --reporter xunit:build2/reports/junit/testcafe-results.xml",
     "test:testcafe-ci": "run-p -r test:testcafe-setup \"test:testcafe-run\" -- 2>/dev/null",
+    "test:tsd": "tsd",
```

Comment:
Is this being run as a part of the build?

Reply:
I've added this scipt to travis: https://github.com/okta/okta-signin-widget/pull/1740/files#diff-6ac3f79fc25d95cd1e3d51da53a4b21b939437392578a35ae8cd6d5366ca5485R22

Reply:
Shall enable for bacon as well. (check the `.bacon.yml`)